### PR TITLE
Wizard, AddressBook, SettingsNode: prevent undesired text selection

### PR DESCRIPTION
**Pull request by xiphon**
Closes https://github.com/monero-project/monero-gui/issues/2497

### CODE REVIEW

File: pages/settings/SettingsNode.qml [hunk 1]
```diff
@@ -232,14 +229,11 @@ Rectangle{
                     font.pixelSize: 15
```
Comment (selsta) [inline pages/settings/SettingsNode.qml]: did this stay TextArea on purpose?

Comment (xiphon) [reply pages/settings/SettingsNode.qml]: Nope, missed this one. Thanks

**Comment by selsta** [comment]
2019-11-27 12:29:53.176	W qrc:/pages/settings/SettingsNode.qml:230 Cannot assign to non-existent property "activeFocusOnPress"

**Comment by xiphon** [comment]
Updated

**Pull request merged**